app/lib/actions/katello/product/update.rb

```diff
@@ -9,9 +9,10 @@ def plan(product, product_params)
             plan_action(::Actions::Katello::Product::RepositoriesGpgReset, product)
           end
 
-          if ::SETTINGS[:katello][:use_cp] && product.productContent_changed?
-            plan_action(::Actions::Candlepin::Product::Update, product)
-          end
+          # what's the deal with this
```

Comment:
This is why I feel the Actions::Candlepin::Product::Update action can be removed - what could make this if statement == true?

Reply:
the only place i can see this being possibly used is if Product.import_from_cp is called prior to this, however it doesn't make much sense to me +1 to remove it, unless we discover something during testing :)